sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuTransitionOverrides.scala

```diff
@@ -34,13 +35,53 @@ import org.apache.spark.sql.rapids.GpuFileSourceScanExecBase
 class GpuTransitionOverrides extends Rule[SparkPlan] {
   var conf: RapidsConf = null
 
-  def optimizeGpuPlanTransitions(plan: SparkPlan): SparkPlan = plan match {
+  def optimizeGpuPlanTransitions(plan: SparkPlan): SparkPlan = {
+
```

Comment:
nit, extra line.

Reply:
So I think if we revert the `plan match {` change, there is no diff in `optimizeGpuPlanTransitions`, I could be missing something.